Applications/Resnet/jni/cifar_dataloader.h

```diff
@@ -107,6 +108,17 @@ class Cifar100DataLoader final : public DataLoader {
   void next(float **input, float **label, bool *last);
 
 private:
+  inline static constexpr int ImageSize = 3 * 32 * 32;
+  inline static constexpr int NumClass = 100;
+  inline static constexpr int SampleSize =
+    1 + 1 + 3 * 32 * 32; /**< 1 coarse label, 1 fine label, pixel size */
+
```

Comment:
What do you mean coarse label and fine label? Could elaborate little bit?

Reply:
Below raw format is used, cited from https://www.cs.toronto.edu/~kriz/cifar.html.

We are only using fine label instead of coarse label though :)
```
Binary version
The binary version of the CIFAR-100 is just like the binary version of the CIFAR-10, except that each image has two label bytes (coarse and fine) and 3072 pixel bytes, so the binary files look like this:
<1 x coarse label><1 x fine label><3072 x pixel>
...
<1 x coarse label><1 x fine label><3072 x pixel>
```